elfpy/data/convert_data.py

```diff
@@ -267,43 +293,73 @@ def get_wallet_info(
                     tokenValue=num_base_token,
                 )
             )
-        # Query and add hyperdrive tokens to walletinfo
+
+        # Query and add LP tokens to wallet info
+        lp_token_prefix = int(hyperdrive_interface.AssetIdPrefix.LP)
+        lp_token_id = hyperdrive_interface.encode_asset_id(lp_token_prefix, 0)
```

Comment:
Can you be explicit about maturity_time=0 here? Maybe also add a comment saying `#  LP tokens & withdrawal shares always have maturity_time=0`. that way future people won't be confused about what is going on.